### PR TITLE
Updating the rancher-windows-exporter chart to use windows prefix path.

### PR DESCRIPTION
**Pull request by phillipsj**
This part of the fix for the bug found [here](https://github.com/rancher/dashboard/issues/2837) with the windows prefix path.

Related Issue: https://github.com/rancher/rancher/issues/32194

### CODE REVIEW

**Review by nickgerace — DISMISSED** [review]

File: packages/rancher-windows-exporter/charts/templates/daemonset.yaml [hunk 1]
```diff
@@ -45,7 +45,7 @@ spec:
         - name: wins-pipe
           mountPath: \\.\pipe\rancher_wins
         - name: binary-host-path
-          mountPath: c:/host/etc/windows-exporter
+          mountPath: {{ regexReplaceAll ".:/" .Values.global.cattle.rkeWindowsPathPrefix  "${1}C:/host/"}}etc/windows-exporter
```
Comment (aiyengar2) [inline packages/rancher-windows-exporter/charts/templates/daemonset.yaml:48]: No need to do the `regexReplaceAll` here; `C:/host/etc/windows-exporter` can be hard-coded.

This is the same logic for why we didn't need to modify the [mountPath in rancher-wins-upgrader](https://github.com/rancher/wins/blob/4bf76f4ecbc0ea7b835a4080d36705737a41559a/charts/rancher-wins-upgrader/templates/daemonset.yaml#L40); this path is just what the **container** sees (and what the scripts reference).

All that needs to be changed is the directory that this containerPath points to; e.g. the hostPath modified below.

File: packages/rancher-windows-exporter/charts/templates/daemonset.yaml [hunk 2]
```diff
@@ -66,7 +66,7 @@ spec:
           path: \\.\pipe\rancher_wins_proxy
       - name: binary-host-path
         hostPath:
-          path: c:/etc/windows-exporter
+          path: {{ .Values.global.cattle.rkeWindowsPathPrefix }}etc/windows-exporter
```
Comment (aiyengar2) [inline packages/rancher-windows-exporter/charts/templates/daemonset.yaml:69]: Looks good, but can you add validation of the rkeWindowsPathPrefix value in this PR too?

e.g. add [`{{ include "winsUpgrader.validatePathPrefix" . }}`](https://github.com/rancher/wins/blob/4bf76f4ecbc0ea7b835a4080d36705737a41559a/charts/rancher-wins-upgrader/templates/configmap.yaml#L1) to the top of this file and add that function to the [`_helpers.tpl`](https://github.com/rancher/wins/blob/4bf76f4ecbc0ea7b835a4080d36705737a41559a/charts/rancher-wins-upgrader/templates/_helpers.tpl#L32-L39).

**Review by aiyengar2 — CHANGES_REQUESTED** [review]

File: packages/rancher-windows-exporter/charts/templates/daemonset.yaml (hunk 2 above)
Comment (phillipsj) [reply packages/rancher-windows-exporter/charts/templates/daemonset.yaml:70]: Done

**Review by aiyengar2 — APPROVED** [review]
LGTM! 😄

**Comment by aiyengar2** [comment]
@phillipsj CI is failing because we didn’t bump the rcVersion of Monitoring V2 as well.

Can you bump the releasedCandidateVersion in packages/rancher-monitoring/package.yaml as part of this change as well?

**Comment by phillipsj** [comment]
@aiyengar2 I have bumped that version.

**Comment by luthermonson** [comment]
https://github.com/rancher/rancher/issues/32194

**Review by aiyengar2 — APPROVED** [review]

**Review by nickgerace — APPROVED** [review]

**Pull request merged**